app/migrations/seed.py

```diff
@@ -257,32 +258,32 @@ def update_or_create_menu(name, description, parent_id=None):
     print('------------------------------------------------------------')
     print('Done creating USSD Menus')
 
+    
+    
+    print('Creating Business Categories')
+    print('**********************************************************************')
+    business_categories = [
+        {'name': 'Food', 'icon': 'message'},
+        {'name': 'Water', 'icon': 'message'},
+        {'name': 'Energy', 'icon': 'message'},
+        {'name': 'Education', 'icon': 'message'},
+        {'name': 'Health', 'icon': 'message'},
+        {'name': 'General shop', 'icon': 'message'},
+        {'name': 'Environment', 'icon': 'message'},
+        {'name': 'Transport', 'icon': 'message'},
+        {'name': 'Labour', 'icon': 'message'},
+        {'name': 'Other', 'icon': 'message'},
+    ]
+    for business_category in business_categories:
+        try:
+            usage = TransferUsage.create_without_duplicate(
```

Comment:
Does it matter if you use functions from the app in a seed script?

Reply:
i think that's generally a totally fine idea...